.github/workflows/release.yml

```diff
@@ -40,7 +37,7 @@ jobs:
       - name: Build Python distributable for ${{ matrix.os }}
         run: |
           cd device-identifier
-          pyinstaller device_identifier_server.py
+          pyinstaller cli.py
```

Comment:
The built python distributable will not work for cli.py.

Reply:
Done